src/CookiePolicy/App.svelte

```diff
@@ -0,0 +1,89 @@
+<svelte:options tag="tree-popup" immutable={true} />
+
+<script>
+    import { writable } from "../../utils/localstorage";
+    let cookiePolicy = localStorage.getItem("hidePlanetCookie");
+    export let message;
+    export let buttontext;
+    export let theme = "light";
+    export const hideCookieNotification = writable(cookiePolicy);
+
+    function hideCookieNotice() {
+        hideCookieNotification.subscribe((value) => {
+            localStorage.setItem("hidePlanetCookie", (value = true));
+        });
+        cookiePolicy = true;
+    }
+</script>
+
+{#if !cookiePolicy}
+    <div
+        class="cookiePolicy"
+        style="--bg-color: {theme === 'light' ? '#fff' : '#2f3336'};
+            --text-color: {theme === 'light'
+            ? '#000000'
+            : '#fff'};"
+    >
+        <div class="policyNotice">
+            <div class="cookiePolicyLink" />
+            {@html message}
+        </div>
+        <button
+            id={"cookieCloseButton"}
+            class="primaryButton"
+            on:click={hideCookieNotice}
+        >
+            {buttontext}
+        </button>
+    </div>
+{/if}
+
+<style>
+    @import "https://fonts.googleapis.com/css2?family=Raleway:wght@400;700&display=swap";
```

Comment:
Since Fonts are already loaded on the page, we should avoid reloading Fonts.  Please check embed in staging if widget works without importing fonts.